jwst/tweakreg/tests/test_tweakreg.py

```diff
@@ -21,6 +29,41 @@ def dummy_source_catalog():
     return catalog
 
 
+@pytest.mark.parametrize("inplace", [True, False])
+def test_rename_catalog_columns(dummy_source_catalog, inplace):
+    renamed_catalog = tweakreg_step._rename_catalog_columns(dummy_source_catalog)
+
+    # if testing inplace, check the input catalog
+    if inplace:
+        catalog = dummy_source_catalog
+    else:
+        catalog = renamed_catalog
+
+    assert 'xcentroid' not in catalog.colnames
+    assert 'ycentroid' not in catalog.colnames
+    assert 'y' in catalog.colnames
+    assert 'y' in catalog.colnames
+
+
+@pytest.mark.parametrize("missing", ["x", "y", "xcentroid", "ycentroid"])
+def test_rename_catalog_columns_invalid(dummy_source_catalog, missing):
+    # if the column we want to remove is not in the table, first run
+    # rename to rename columns this should add the column we want to remove
+    if missing not in dummy_source_catalog.colnames:
+        tweakreg_step._rename_catalog_columns(dummy_source_catalog)
+    dummy_source_catalog.remove_column(missing)
+    with pytest.raises(ValueError, match="catalogs must contain"):
+        tweakreg_step._rename_catalog_columns(dummy_source_catalog)
+
+
+def test_rename_catalog_columns_inplace(dummy_source_catalog):
```

Comment:
How does this test relate to the test on line [33](https://github.com/spacetelescope/jwst/pull/8424/files#diff-38c152a7d4980762e744808c1ab3e526002f7eff5d433762d1867f99e1c54b8bR33), is it the same as part of `test_rename_catalg_columns`?

Reply:
Thanks for finding this duplicate. I removed it in: 0681bb5c9